website/docs/r/subscription_feature.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Base"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_subscription_feature"
+description: |-
+  Manages a Feature.
+---
+
+# azurerm_subscription_feature
+
+Register and unregister a preview feature for the subscription. Features which `approvalType` is `AutoApproval` can be managed by terraform. 
```

Comment:
Can we provide a link to a doc here to help users with what they can manage, or how they can find out if their desired feature is auto or manual approval?